spyderlib/utils/icon_manager.py

```diff
@@ -181,10 +183,16 @@
     '1uparrow':                [('fa.angle-up',), {}],
     '2downarrow':              [('fa.angle-double-down',), {}],
     '1downarrow':              [('fa.angle-down',), {}],    
-    'pylint':                  [('fa.search', 'fa.check'), {'options': [{}, {'offset': (0.125, 0.125), 'color': 'orange'}]}],
     'attribute':               [('spyder.circle-letter-a',), {'color': 'magenta'}],
     'module':                  [('spyder.circle-letter-m',), {'color': '#daa520'}],
     'no_match':                [('fa.circle',), {'color': 'gray'}],
+    'no_match':                [('fa.circle',), {'color': 'gray'}],
+    # --- Third party plugins ------------------------------------------------
+    'profiler':                [('fa.clock-o',), {}],
+    'pylint':                  [('fa.search', 'fa.check'), {'options': [{}, {'offset': (0.125, 0.125), 'color': 'orange'}]}],
+    'condapackages':           [('fa.archive',), {}],
+    'spyplugins.ui.example':   [('fa.eye',), {}],
+    'spyplugins.ui.autopep8':  [('fa.eye',), {}],
```

Comment:
Two things:

1. How are we going to handle this icon assignment for plugins written by people who are not Spyder developers?
2. Please rename the last two plugins in this list with the names we previously agreed :-) (in their repos and here too).